syntax = "proto3";

message LocationMessage {
  int32 person_id = 1;
  string creation_time = 2;
  string longitude = 3;
  string latitude = 4;
}

message PersonMessage {
    string first_name = 1;
    string last_name = 2;
    string company_name = 3;
}


service ApiService {
  rpc create_location(LocationMessage) returns (LocationMessage);
  rpc create_person(PersonMessage) returns (PersonMessage);
}